pyvista/themes.py

```diff
@@ -2090,6 +2094,8 @@ def __repr__(self):
             'Return Camera Position': 'return_cpos',
             'Hidden Line Removal': 'hidden_line_removal',
             'Anti-Aliasing': '_antialiasing',
+            'Split sharp edges': '_split_sharp_edges',
+            'Sharp edges feature angle': '_sharp_edges_feature_angle',
```

Comment:
This new name is one longer than the longest previous name:
```
>>> pv.global_theme
[snip]
Return Camera Position   : None
Hidden Line Removal      : False
Anti-Aliasing            : False
Split sharp edges        : False
Sharp edges feature angle: 30.0
```
We should consider adding one more space, i.e. changing `'<25'` to `'<26'` on the loop right below. Do you think this can break anything, or should this be safe?

It would be best to compute that length dynamically, but for that we'd probably have to recurse over the whole tree and find the longest length first...

Reply:
Good point.

What I've done is added a test for this in 1bf8cd4b2911b4c767181f355747ce7a953e3013 and shortened the length of the key. I could have lengthened everything, but I think the abbreviation is obvious.